src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,93 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static StorageDao storageDao = new StorageDaoImpl();
+
+    @BeforeAll
+    static void setUp() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @AfterEach
+    void clearList() {
+        if (!Storage.people.isEmpty()) {
+            Storage.people.clear();
+        }
+    }
+
+    @Test
+    void checkEmptyList() {
+        User testUser1 = new User("loginn", "password", 18);
+        registrationService.register(testUser1);
+        assertFalse(Storage.people.isEmpty());
+    }
+
+    @Test
+    void actualListSizeAndContent() {
```

Comment:
The test method name `actualListSizeAndContent` does not follow the naming convention. A better name could be `register_multipleUsers_storageContainsAllUsers`.

Reply:
Agree